pkg/syncer/shared/namespace.go

```diff
@@ -31,14 +35,25 @@ const (
 // NamespaceLocator stores a logical cluster and namespace and is used
 // as the source for the mapped namespace name in a physical cluster.
 type NamespaceLocator struct {
-	LogicalCluster logicalcluster.Name `json:"logical-cluster"`
-	Namespace      string              `json:"namespace"`
+	Workspace          logicalcluster.Name `json:"workspace,omitempty"`
+	WorkloadCluster    logicalcluster.Name `json:"workloadCluster,omitempty"`
+	WorkloadClusterUID types.UID           `json:"workloadClusterUID,omitempty"`
```

Comment:
WorkloadCluster will become SyncTarget in some days.

Reply:
a structure like `{"syncTarget": {"path": ...,"name": ...,"uid", ...}, "workspace": ..., "namespace": ...}` would make that read much easier.